docs/CaliptraHardwareSpecification.md

```diff
@@ -1299,17 +1299,27 @@ In practice, observing a t-value greater than a specific threshold (mainly 4.5)
 ![](./images/TVLA_threshold.png)
 
 
+##### KeyGen TVLA
+
+We detected a leakage using TVLA in the HMAC_DRBG algorithm during ECC key generation, based on 150,000 power traces. The leakage originated from a part of the SHA512 function (w_data) that was not fully protected by masking. Same leakage is expected for HMAC operations. 
```

Comment:
is `Same leakage` accurate or was this intended to be `Some leakage`? If `Same leakage` is accurate, then recommend update to `The same leakage`

Reply:
Done!